src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/utils/secretmanager_constants.py

```diff
@@ -1,2 +1,6 @@
 # Used as secretsmanager key for CircleCI Api key
```

Comment:
```python
# Secrets Manager secret name, whose value is the
# CircleCI API key used by the Lambda credential rotator
# to update environment variables.
```

Unless this is the API key used by CircleCI itself to update environment variables during the build/test/deploy process?

Reply:
Updated